### PR TITLE
Assign multiple people to an activity

### LINKED ISSUE #118: Add activity that includes multiple people
Feedback from the community ([source](https://news.ycombinator.com/reply?id=14507921&goto=item%3Fid%3D14497295%2314507921)):

It would be great if there would be a place to set up a new activity that includes multiple people. So entering a birthday party with 15 people is a bit more practical than it is now. I first thought that's what the journal is about, but it doesn't seem have this feature (yet?). Like it is now, the journal seems not very useful at all to be honest.

### PR DESCRIPTION
**Pull request by mheap**
This resolves #118 by adding the ability to assign an activity to multiple people. I feel that activities are a better place for this than the journal. This PR includes two migrations - one to create an `activity_contact` pivot table and another to migrate the existing data in to it. I did not remove the old `contact_id` field from `activities` yet as it would destroy data. We can add another migration in the future to remove it once this change is proven to work.

I reused the search bar from the top and refactored the javascript a little so that it can be applied in multiple places on a page. The contacts attached to an activity can be clicked to remove them. I didn't commit an updated `app.js`, just the edited `vanilla.js`.

There is one small issue, where you get redirected to the newest contact after adding/updating an activity. The best way to resolve this would be to build an activities screen that we can link to for a specific activity.

Here's a gif of it working:

If there are any suggested changes, let me know and I'll see what I can do

cc @djaiss 

### Checklist

- [x] Read the [CONTRIBUTING document](https://github.com/monicahq/monica/blob/master/CONTRIBUTING.md) before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [x] Make sure that the change you propose is the smallest possible.
- [x] Screenshots are included if the PR changes the UI.
- [x] Tests added for this feature/bug.
- [x] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG) entry added, if necessary, under `UNRELEASED`.
- [x] [CONTRIBUTORS](https://github.com/monicahq/monica/blob/master/CONTRIBUTORS) entry added, if necessary.
- [x] Indicate `[wip]` in the title of the PR it is is not final yet. Remove `[wip]` when ready. Otherwise the PR will be considered complete and rejected if it's not working.

### CODE REVIEW

**Comment by djaiss** [comment]
This is incredible.

Thanks so much for your time. I'll review this PR in a couple hours.

File: resources/lang/en/people.php [hunk 1]
```diff
@@ -258,6 +258,7 @@
     'activities_add_success' => 'The activity has been added successfully',
     'activities_update_success' => 'The activity has been updated successfully',
     'activities_delete_success' => 'The activity has been deleted successfully',
+    'activities_who_was_involved' => 'Who was involved?',
```
Comment (djaiss) [inline resources/lang/en/people.php]: Could you also add this key to the other languages files (you can copy/paste the EN translation, but at least the key will be there for other translators to translate it)?

**Comment by djaiss** [comment]
Left two comments above! Otherwise, 👍 so far, it's 🌟 !!

**Comment by mheap** [comment]
@djaiss I'm new to how Laravel builds JS, so I'm not sure what lives where. I've added the code to `resources/assets/js/search.js`, run `gulp` and committed the generated `vanilla.js`. I hope this is enough.

I've also fixed the translations by adding the key to all existing languages. I used Google Translate to try and get the text correct for each language.

**Comment by mheap** [comment]
@djaiss I've added translations for everything I could see (including dashboard events + added a new icon). If there's anything else let me know and I'll get it resolved this weekend

**Comment by djaiss** [comment]
Thanks a lot for your changes! I'm going to make the final review now and see if everything's good!

**Comment by djaiss** [comment]
Thanks a lot for all your hard work!

**Pull request merged**